edina/src/data/Su/json/about.json

```diff
@@ -0,0 +1,172 @@
+        "content": [
+            "苏星    苏星，中国美协漆画艺术委员会委员、中国美术家协会会员、广东省美术家协会理事、广州市美术家协会副主席、广东省美协漆画艺委会主任、广州市美协漆画艺委会主任、岭南漆画学会会长、国家二级美术师、广州美术学院客座副教授、广州海珠区政协委员。\r    1962年生于广东佛山，1985年毕业于广州美术学院漆画专业。1986年与七名画家组成“小草画屋”，在法国、日本、新加坡、深圳、广州展示其新颖独特的现代装饰画，轰动一时；2006年其漆画作品《海内存知己 天涯若比邻》获中国礼宾司批准，作为政府礼物赠送给随哥德堡号来穗的瑞典国王夫妇，使中国漆画备受关注；2009年策划组织“岭南漆画世界巡回展”，带领岭南漆画首次跨出国门、走向世界，再次引起轰动，相关报道经国内外50多家媒体转载；2010年为广州城隍庙创作大型漆壁画《开天辟地•神仙卷》，并再次赴德国法兰克福举办“自然•生命”苏星、刘春潮漆画作品联展；2012年应邀到美国纽约采风、创作、举办个展，到日本交流；2013年到澳大利亚交流、第二次参加“艺术广东”大型国际艺术展、接连策划组织“广东省第三届漆画作品展”、“漆重奏——广东省漆画精品展”； 2014年4月、5月分别在广州荣宝斋与国家画院美术馆深圳坪山分馆成功举办“星光漆彩——乔十光•苏星漆画联展”、“岁月如歌——苏星漆画作品展”，这是荣宝斋、国家画院美术馆首次举办漆画展；2014年6月与当代五弦琵琶代表人物方锦龙先生联袂创作的仿唐五弦漆琵琶“唐韵”被国家大剧院珍藏，成为大剧院艺术宝库中第一把集演奏、观赏和收藏为一体的珍品乐器；2014年10月受邀担任“2014第六届新西兰读书文化节”文化大使，分别在奥克兰、惠灵顿举办“漆画中国——苏星新西兰漆画展”。这也是中国漆画首次走进新西兰，受到观众热捧、媒体连续报道……2015年1月参加2015洛杉矶艺术展（Los Angeles Art Show），两幅作品被美国藏家收藏。2015年9月在日本轮岛漆艺美术馆策划举办广东岭南漆画展，受到前史雄、田畸昭一郎等日本漆艺大师的好评。\r    苏星长期潜心研究、创作、推广漆画，不断尝试新技法、探索新路向、吸收新思维，成绩斐然，作品多次在国内各类美展中获奖、在国外展出，多次作为全国评委作品受邀参加全国美展，其本人也多次作为全国评委参加全国美展评选。2005年更创造性研制出管状的漆画颜料，被行内专家誉为“漆画史上的一大创新”、“漆画发展的历史性突破”。2006年获广州市文联、市美协联合颁发的“城市美术贡献”奖，2008年获广州市委市政府联合颁发的“第六届广州文艺奖”三等奖，2010、2011连续两年获广州市美协颁发的“工作优异”嘉奖证书。",
+            "Born in 1962 at Foshan Guangdong, Su Xing(Han nationality) graduated at Guangzhou Academy of Fine Arts in 1985 majoring in lacquer painting, and founded “Little Grass Studio” with 7 other artists, whose unique modern decorative paintings made a hit in France, Japan, Singapore, Shenzhen China and Guangzhou China.\r　　In 2006, after approved by China Protocol Department, the artwork of Mr.Su- Long Distance Separates No Bosom Friends was given to the King of Sweden as official present, who came to Guangzhou by East Indiaman Gotheborg. 　　\r　　In 2009, Mr. Su planed and organized “Lingnan* Lacquer Painting World Exhibition Tour”, it is the first time the Lingnan lacquer painting unveiled its beauty to the world, which had caught the great attentions of more than 50 medias. \r　　He is an accomplished lacquer painter. Researching with great concentration of long-term, creating with unlimited enthusiasm, promoting with inexhaustible vigor, absorbing new techniques, Mr. Su’s artworks have got numerous prizes in China and exhibited abroad. Moreover, he has created tubular lacquer pigment, praised highly by professionals in the circle. With all these achievements, Mr. Su got Urban Art Contribution prize awarded by Guangzhou Federation of Literary and Art Circles and Guangzhou Fine Art Society in 2006；in 2008, he got the 6th Guangzhou Literary and Art Prize awarded by Guangzhou Municipality. 　　\r　　At present, Su is one member of China Lacquer Painting Society and China Artists Association (CAA), deputy chairman of Guangzhou Artists Association (GAA), director of the Association’s Lacquer Painting Art Committee, GAA, chairman of Lingnan Lacquer Painting Society, China 2nd-Grade Artist, and guest professor of Guangzhou Academy of Fine Arts."
+        ]
+    },
+    {
+        "title": "chronology",

```

Comment:
We will need two set of data for both English and Chinese. 
If data is missing, put: MISSING DATA
If translation is missing, put: MISSING TRANSLATION
if something else, put: TBD